pkg/operator/object/ceph/rgw.go

```diff
@@ -349,7 +349,7 @@ func rgwContainer(store rookalpha.ObjectStore, version string) v1.Container {
 			"rgw",
 			fmt.Sprintf("--config-dir=%s", k8sutil.DataDir),
 			fmt.Sprintf("--rgw-name=%s", store.Name),
-			fmt.Sprintf("--rgw-host=%s", instanceName(store)),
+			fmt.Sprintf("--rgw-host=%s", fmt.Sprintf("%s.%s", instanceName(store), store.Namespace)),
```

Comment:
shouldnt this be in the form of `<service name>.<namespace>`? It seems `instanceName(store)` gives you the the name in the form of `<appName>-<name>`. See https://github.com/rook/rook/blob/master/pkg/operator/object/ceph/rgw.go#L258

Reply:
`instanceName(store)` gives you the name of the service. So the result is that we append the namespace to give you the dns name in case you are outside the namespace

Reply:
OK LGTM